lib/rush/game.ex

```diff
@@ -15,9 +15,9 @@ defmodule Rush.Game do
 
     {:ok, _game_logger} = GenServer.start_link(Rush.GameLogger, [])
 
-    {:ok, _} = GenStage.sync_subscribe(segment1, to: spawner)
-    {:ok, _} = GenStage.sync_subscribe(segment2, to: spawner)
-    {:ok, _} = GenStage.sync_subscribe(segment3, to: spawner)
+    {:ok, _} = GenStage.sync_subscribe(segment1, to: spawner, max_demand: 10, min_demand: 4)
```

Comment:
@seangwright This was the change needed.  The segments were aggressively buffering the monsters.  The default max demand is like 1k so that's why we briefly saw the spawner with some monsters queued after letting it run for a while.